src/object-mirrors.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+var ERROR_SOURCE_PROPERTY_NOT_IMPLEMENTED = 'Requiring undefined property on source object';
```

Comment:
This error message does not seem adapted to the `checkProperty` call in `set` in `mirrorRwProp`